multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -273,10 +273,13 @@ def is_installed(self, tag: str) -> bool:
         return all(dependency is not None for dependency in dependencies)
 
     def install(self, overwrite: bool) -> None:
+        self._check_install_env()
```

Comment:
I think that if someone uses the `--overwrite` flag, the existing `CARGO_HOME` and `RUSTUP_HOME` should reset to the default location, as if `rust` was not previously installed.

Reply:
Indeed, we should do that :raised_hands:

Reply:
Fixed.